pkgs/top-level/all-packages.nix

```diff
@@ -11153,6 +11153,8 @@ with pkgs;
 
   ophis = python3Packages.callPackage ../development/compilers/ophis { };
 
+  ophcrack-cli = ophcrack.override { enableGui = false; };
```

Comment:
So `c` is behind `i`... And how old is your local checkout?

Reply:
5mins